src/Fixer/FunctionNotation/FunctionDeclarationFixer.php

```diff
@@ -24,11 +26,53 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class FunctionDeclarationFixer extends AbstractFixer
+final class FunctionDeclarationFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
+    const SPACING_NONE = 'none';
```

Comment:
Not sure about adding these (public) `const`'s here, for the other fixers we didn't do that either.
Maybe we could add those `WhitespacesFixerConfig` and re-use over other fixers? (don't do that in this PR though for now, lets discuss first ;) ), otherwise I would opt for removing and just using the hardcoded string.